### PR TITLE
feat(electron): Update docs for v5

### PR DESCRIPTION
**Pull request by timfish**
This is a draft until v5 stable is released!

### CODE REVIEW

File: docs/platforms/javascript/guides/electron/configuration/integrations/electronbreadcrumbs.mdx [hunk 1]
```diff
@@ -5,19 +5,19 @@ description: "Supports capturing events from `uncaughtException` while retaining
 sidebar_order: 10
 ---
 
-Captures breadcrumbs for events for many of Electron's built-in modules. The breadcrumbs captured for each module can be configured `true` to capture all events or `false` to capture no events. Alternatively, you can supply a function which is passed the event name and returns `true` or `false`, depending on whether the event should be captured.
+Captures breadcrumbs for many of Electron's built-in modules. The breadcrumbs captured for each module can be configured `true` to capture all events or `false` to capture no events. Alternatively, you can supply a function which is passed the event name and returns `true` or `false`, depending on whether the event should be captured.
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/electron/configuration/integrations/electronbreadcrumbs.mdx:8]:
```suggestion
This integration captures breadcrumbs for many of Electron's built-in modules. The breadcrumbs captured for each module can be configured as `true` to capture all events, or `false` to capture no events. Alternatively, you can supply a function which is passed the event name and returns `true` or `false`, depending on whether the event should be captured.
```

File: docs/platforms/javascript/guides/electron/configuration/integrations/mainprocesssession.mdx [hunk 2]
```diff
@@ -7,14 +7,17 @@ sidebar_order: 10
 
 Captures sessions linked to the lifetime of the Electron main process. Capturing sessions makes it possible to show [release health](/product/releases/health/) statistics in Sentry.
 
-If you'd prefer to capture sessions linked to the focus of `BrowserWindow`'s, use the <PlatformLink to="/configuration/integrations/browserwindowsession">BrowserWindowSession</PlatformLink> integration.
+If you'd prefer to capture sessions linked to the focus of `BrowserWindow`'s,
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/electron/configuration/integrations/mainprocesssession.mdx:10]:
```suggestion
If you'd prefer to capture sessions linked to the focus of `BrowserWindow`,
```

File: docs/platforms/javascript/guides/electron/configuration/integrations/net.mdx [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Net
+title: ElectronNet
 excerpt: ""
 description: "Captures breadcrumbs and tracing spans for Electrons `net` module. (default)"
 sidebar_order: 10
 ---
 
 Captures breadcrumbs and tracing spans for Electron's `net` module. Breadcrumbs and tracing spans are enabled by default, while `sentry-trace` headers are added to requests for all origins.
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/electron/configuration/integrations/net.mdx:8]:
```suggestion
This integration captures breadcrumbs and tracing spans for Electron's `net` module. Breadcrumbs and tracing spans are enabled by default, while `sentry-trace` headers are added to requests for all origins.
```

File: docs/platforms/javascript/guides/electron/configuration/integrations/net.mdx [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 
-The integration can be configured via constructor options:
+The integration can be configured via options:
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/electron/configuration/integrations/net.mdx:10]:
```suggestion
The integration can be configured with the following options:
```

File: docs/platforms/javascript/guides/electron/configuration/integrations/additionalcontext.mdx [hunk 5]
```diff
@@ -7,14 +7,11 @@ sidebar_order: 10
 
 Adds additional device context to events.
 
-The constructor takes a number of options which all default to `true`:
+The integration takes a number of options which all default to `true`:
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/electron/configuration/integrations/additionalcontext.mdx:10]:
```suggestion
This integration takes a number of options, which all default to `true`:
```

File: docs/platforms/javascript/guides/electron/configuration/integrations/mainprocesssession.mdx [hunk 6]
```diff
@@ -7,14 +7,17 @@ sidebar_order: 10
 
 Captures sessions linked to the lifetime of the Electron main process. Capturing sessions makes it possible to show [release health](/product/releases/health/) statistics in Sentry.
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/electron/configuration/integrations/mainprocesssession.mdx:8]:
```suggestion
This integration captures sessions linked to the lifetime of the Electron main process. Capturing sessions makes it possible to show [release health](/product/releases/health/) statistics in Sentry.
```

File: docs/platforms/javascript/guides/electron/configuration/integrations/preloadinjection.mdx [hunk 7]
```diff
@@ -5,13 +5,17 @@ description: "Injects a preload script via the Electron. (default)"
 sidebar_order: 10
 ---
 
-Injects a preload script via the Electron [`session.setPreloads(preloads)`](https://www.electronjs.org/docs/latest/api/session#sessetpreloadspreloads) API. By default, sessions are only injected into the `defaultSession`. If you're using other sessions, you can pass a function as `getSessions` to `init`.
+Injects a preload script via the Electron
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/electron/configuration/integrations/preloadinjection.mdx:8]:
```suggestion
This integration injects a preload script via the Electron
```

**Review by vivianyentran — APPROVED** [review]
Added some copyedits to the content changes

**Review by AbhiPrasad — APPROVED** [review]

**Pull request merged**